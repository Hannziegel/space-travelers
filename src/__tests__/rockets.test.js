import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import RocketsList from '../components/RocketList';

const renderRocketList = () => {
  const { container } = render(
    <Provider store={store}>
      <RocketsList />
    </Provider>,
  );
  return container;
};

describe('Test Rocket page', () => {
  test('Match Snapshot & render Rocket List', () => {
    expect(renderRocketList().firstChild).toMatchSnapshot();
  });
});
