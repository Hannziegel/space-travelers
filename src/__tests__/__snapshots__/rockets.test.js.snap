// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Rocket page Match Snapshot & render Rocket List 1`] = `null`;
